File: issta2018artifacts/minimized-bugs/buggy-clousot/PlumeLib/NoExit.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

// code that causes the bug fixed in plume-lib by rev. 95b3cab

public class NoExit
{
    public static void Main(string[] args)
    {
        if (args.Length != 2)
        {
            Console.Error.WriteLine("Needs 2 arguments, got " + args.Length);
        }
        //:: error: (array.access.unsafe.high.constant)
        int limit = int.Parse(args[0]);
    }
}

File: issta2018artifacts/minimized-bugs/buggy-key/NoExit.java
// code that causes the bug fixed in plume-lib by rev. 95b3cab

public class NoExit {

    /*@ public normal_behavior
@ requires true;
@ ensures true;
@*/
     public static void main(String[] args) {
        if (args.length != 2) {
          println("Needs 2 arguments, got " + args.length);
        }
	//:: error: (array.access.unsafe.high.constant)
        String limit = args[0];
     }

    /*@ public normal_behavior
@ requires true;
@ ensures true;
@*/
    private static void println(String s) {
    }
}
